docs/FAQ.rst

```diff
@@ -107,13 +107,58 @@ https://github.com/hyperledger/avalon/issues
 How many nodes does Avalon support?
 -----------------------------------
 One, specifically the Avalon blockchain connector that connects to
-a blockchain node.  In the future Avalon will support processing
+a blockchain node. In the future Avalon will support processing
 multiple work orders in parallel, but there will still be one connector.
 No other Avalon component connects to the blockchain.
 
 Another Avalon instance unrelated to any other Avalon instance may connect
 independently to the blockchain.
 
+What synchronization modes does Avalon Support?
+-----------------------------------------------
+The Off-Chain Trusted Compute Spec has three modes (asynchronous,
+notify, and synchronous). Avalon currently supports async mode.
+Synch mode may be implemented, depending on interest.
+
+Is the LMBD database available for application use?
+---------------------------------------------------
+No, that is not the intention. LMDB is dedicated for internal use by Avalon.
+Application data can be stored in a separate database or in files.
+The only potential external use that should be made of LMDB is by an
+Avalon hosting service for its monitoring and administration tools.
+
+Is there a mechanism to auto-trim the LMDB database?
+----------------------------------------------------
+Avalon is supposed to auto-trim LMDB after exceeding a maximum threshold
+by removing old entries. It may not be fuly implemented yet,
+but that is the plan. There is also an explicit API to delete a specific
+workorder, so the orchestrator app can explicitly remove completed workorders.
+This is especially useful for to remove large, obsolete responses so as to not
+exhaust the database. This API mitigates the LMDB size but is not intended as
+a replacement for auto-clean up.
+
+In the future, a clean up tool may be added.
+
+Does Avalon support a Library OS (LibOS)?
+-----------------------------------------
+Avalon does not support a LibOS now, but support is planned in the future,
```

Comment:
Can the advantage of using a LibOS also be highlighted here?

Reply:
@rranjan3 I added LibOS advantages to the glossary, below.